python/notebooks/glm_demo.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
```

Comment:
I looks like this was generated from Jupyter. Was this checked in on purpose?

Reply:
this script is also unnecessary in this location, so it was removed.